src/platform/backend/drm.c

```diff
@@ -591,10 +592,24 @@ update_outputs(struct chck_pool *outputs)
    return count;
 }
 
+static bool
+use_egldevice(int drmFd)
+{
+   const char* bufferApi = getenv("WLC_BUFFER_API");
+   if (chck_cstr_is_empty(bufferApi)) {
+      drmVersion *version = drmGetVersion(drmFd);
+      bool use_egl = chck_cstreq(version->name, "nvidia-drm");
+      drmFreeVersion(version);
+
+      return use_egl;
+   }
+
+   return !strcasecmp(bufferApi, "egl");
```

Comment:
Then again, I don't mind if this was just case sensitive comparsion since strcasecmp isn't standard function.